ouroboros-network/src/Ouroboros/Network/NodeToNode/Version.hs

```diff
@@ -33,16 +33,22 @@ data NodeToNodeVersion
     -- ^ Changes:
     --
     -- * Enable full duplex connections.
+    | NodeToNodeV_10
```

Comment:
a TODO to follow-up on (this is V_10 in anticipation of the pipelining PR merging?)

Reply:
Yes, it is